aws/cloudformation/ecs/posthog.yaml

```diff
@@ -864,9 +873,57 @@ Resources:
         - Key: Name
           Value: !Ref AWS::StackName
 
+  # Alarms
+  PosthogAlarmTopic:
+    Type: "AWS::SNS::Topic"
+    Properties:
+      DisplayName: "Posthog Error Topic"
+      TopicName: "posthogErrors"
+
+  PosthogErrorEmailSubscription:
+    Condition: HasAlarmEmail
+    Type: 'AWS::SNS::Subscription'
+    Properties:
+      Endpoint: !Ref AlertEmail
+      Protocol: email
+      TopicArn: !Ref PosthogAlarmTopic
+
+  RDSDiskFullAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmDescription: "RDS database is nearly out of storage space - 1GB or less left"
+      AlarmActions:
+      - Ref: PosthogAlarmTopic
+      MetricName: FreeStorageSpace
+      Namespace: AWS/RDS
+      Statistic: Average
+      Period: 600 # 10 minutes
+      Threshold: 1000000000 # Less than 1GB
+      ComparisonOperator: LessThanThreshold
+      EvaluationPeriods: 1
+      OKActions:
+      - Ref: PosthogAlarmTopic
+
+  ElastiCacheMemoryFullAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmDescription: "Redis memory over the last 10 minutes is too low, your instance might not be ingesting events"
+      AlarmActions:
+      - Ref: PosthogAlarmTopic
+      MetricName: FreeableMemory
+      Namespace: AWS/RDS
+      Statistic: Average
+      Period: 600 # 10 minutes
+      Threshold: 640000000 # Less than 64MB
```

Comment:
We have a lot of hobby users, deploying PH for small projects or just to try it out. Would this not trigger for them?

Also, would this not trigger if I set up PostHog and don't ingest events in the first 10min?

Could be my ignorance here as to how low 64mb for redis actually is.

Reply:
So redis runs in memory/in ram. The smallest redis instance available in AWS has 555mb of ram. https://aws.amazon.com/elasticache/pricing/ 

I think this is fair.